lib/web_ui/lib/src/engine/html/render_vertices.dart

```diff
@@ -779,6 +800,14 @@ class _GlContext {
       return canvas;
     }
   }
+
+  /// Returns image data in data url format.
+  String toImageUrl() {
+    html.CanvasElement canvas = html.CanvasElement(width: _widthInPixels, height: _heightInPixels);
```

Comment:
How often is this called? If a lot, we should either cache and reuse the canvas element, or set the width/height to 0 so that we don't run iOS Safari out of memory.

Reply:
Done.

Reply:
I'm not sure what's done :)

Reply:
Oh, nvm, you do set the dimensions to 0